trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/result/ExpressionResultTransformer.java

```diff
@@ -1147,4 +1076,104 @@ ExpressionResult apply(final ExpressionResultTransformer ert, final ExpressionRe
 				final CType targetCType, final ILocation loc, final IASTNode hook);
 	}
 
+	public interface IPointerReadWrite {
```

Comment:
Please document this interface and its methods. It is not clear to me what each of them is supposed to do.

Reply:
Also, just for my understanding: Could you explain how the two classes implementing this interface differ from `HeapLValue` vs `LocalLValue` ? Do we need both distinctions?